src/nuclio/functions/function-collapsing-row/function-collapsing-row.less

```diff
@@ -123,4 +123,14 @@
     .running-status {
         color: @tealish;
     }
+
+    .actions-more-info {
+        position: fixed;
+        z-index: 10;
+    }
+
+    &.invocation-tooltip {
```

Comment:
you used the parent selector to the `ncl-function-collapsing-row` class. That means this class isn't used.